<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>524</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="Btn_Switch_UI">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>界面切换</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QPushButton" name="Btn_Angle_Clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化完成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>位置调整</string>
    </property>
    <widget class="QRadioButton" name="Btn_LM3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Btn_LM1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Btn_LM2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Btn_RM3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机6</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Btn_RM1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Btn_RM2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>电机5</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>192</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="comBox_Serial_num"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Btn_Check_Serial">
       <property name="text">
        <string>检测串口</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Btn_Open_Serial">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Btn_Close_Serial">
       <property name="text">
        <string>关闭串口</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Btn_BackZero">
       <property name="text">
        <string>回零</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Btn_End">
       <property name="text">
        <string>结束</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="Title_all">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>髋关节康复控制系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_to_Curve_Setting">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>曲线配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>测试时间：</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
